code/datums/martial.dm

```diff
@@ -386,19 +389,19 @@
 	usr << "<span class='notice'>Head Kick</span>: Disarm Harm Harm. Decent damage, forces opponent to drop item in hand."
 	usr << "<span class='notice'>Elbow Drop</span>: Harm Disarm Harm Disarm Harm. Opponent must be on the ground. Deals huge damage, instantly kills anyone in critical condition."
 
-//CQC
+//cqc
 #define SLAM_COMBO "GH"
-#define KICK_COMBO "HDH"
+#define KICK_COMBO "HH"
```

Comment:
I'm pretty sure this will conflict with `CONSECUTIVE_COMBO` since that uses HHD, unless that was the intention.